specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -271,7 +271,7 @@
                 {
                   "value": "en-us",
                   "name": "EnUs",
-                  "description": "English"
+                  "description": "English (US)"
```

Comment:
I probably should've brought this up before, but it just occurred to me now: other language references in the swagger (all of them, if memory serves me correctly) are just the language. Only a few languages have culture variations. So is this really "en-us", or should it just be "en"? Does culture (number formatting, etc.) really play a factor here?

Reply:
I believe it does. We will be adding various speller lexicons, many of them being culture-specific.